routes/api/contacts.js

```diff
@@ -1,25 +1,87 @@
-const express = require('express')
+const express = require("express");
 
-const router = express.Router()
+const {
+	listContacts,
+	getContactById,
+	removeContact,
+	addContact,
+	updateContact,
+} = require("../../models/contacts");
+const {
+	newContactValidator,
+	updateContactValidator,
+} = require("../../validators/contactsValidator");
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const router = express.Router();
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/", async (req, res, next) => {
+	const contacts = await listContacts();
+	res.send({
+		contacts,
+	});
+});
 
-router.post('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/:contactId", async (req, res, next) => {
+	const id = req.params.contactId;
+	const contact = await getContactById(id);
+	if (contact) {
+		res.send({
+			contact,
+		});
+	} else {
+		res.status(404).send({
+			message: "Not found",
+		});
+	}
+});
 
-router.delete('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.post("/", async (req, res, next) => {
+	const body = req.body;
+	const { name, email, phone } = req.body;
 
-router.put('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+	const isValid = newContactValidator.validate(body);
 
-module.exports = router
+	if (!name || !email || !phone) {
```

Comment:
newContactValidator.validate <-- tutaj widzę że jest require czyli nie potrzebujesz tego sprawdzenia? 
Ale musisz zwrócić informację, czyli 
`isValid.error` powinn być wcześniej zaraz po linijce 42 i jeśli coś jest nie tak to zatrzymaj operacje `return res.status(400)... wiadomość

Reply:
Chciałam tym sprawić, żeby w przypadku braku, którejś danej pojawiła się widomość missing required name - field, zamiast wiadomości, którą podpowiada biblioteka joi. Rozumiem, że lepiej by było usunąć tą opcję w if i zostawić samo required w walidacji?